src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/SexGenotypeContigPairKey.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.hellbender.tools.exome.germlinehmm;
+
+import org.broadinstitute.hdf5.Utils;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+
+/**
+ * This class provides a key made of the String pair (sex genotype, contig).
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class SexGenotypeContigPairKey implements Serializable {
```

Comment:
Single-use Plain Old Data classes with use-specific names that exist solely for increasing readability are OK, but should perhaps be less visible.  In contrast, I think that classes that represent Plain Old Data that are intended for reuse should have names that are expressive and descriptive, but do not restrict or define use.